File: web/src/main/java/web/web/HomeController.java
package web.web;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;
import io.swagger.v3.oas.annotations.Hidden;

/**
 * Home page controller.
 *
 * @author Paul Chapman
 */
@Controller
@Hidden // Oculta este endpoint en Swagger (es solo HTML, no API REST)
public class HomeController {

    @RequestMapping("/")
    public String home() {
        return "index";
    }

}

File: accounts/src/main/kotlin/web/HomeController.kt
package accounts.web

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.RequestMapping
import io.swagger.v3.oas.annotations.Hidden;

/**
 * Home page controller.
 *
 * @author Paul Chapman
 */
@Controller
@Hidden // Oculta este endpoint en la documentación Swagger (es solo una página HTML)
class HomeController {
    @RequestMapping("/")
    fun home(): String {
        return "index"
    }
}